crawler/tjpb/crawler-tjpb.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/antchfx/htmlquery"
+	"golang.org/x/net/html"
+)
+
+const baseURL string = "https://www.tjpb.jus.br/transparencia/gestao-de-pessoas/folha-de-pagamento-de-pessoal"
+
+var netClient = &http.Client{
+	Timeout: time.Second * 60,
+}
+
+var monthStr = map[int]string{
+	1:  "Janeiro",
+	2:  "Fevereiro",
+	3:  "Março",
+	4:  "Abril",
+	5:  "Maio",
+	6:  "Junho",
+	7:  "Julho",
+	8:  "Agosto",
+	9:  "Setembro",
+	10: "Outubro",
+	11: "Novembro",
+	12: "Dezembro",
+}
+
+func main() {
+	month := flag.Int("mes", 0, "Mês a ser analisado")
+	year := flag.Int("ano", 0, "Ano a ser analisado")
+	flag.Parse()
+	if *month == 0 || *year == 0 {
+		log.Fatalf("Need all arguments to continue, please try again: \"go run crawler-tjpb.go --mes=int --ano=int\"")
+	}
+
+	doc, err := loadURL(baseURL)
+	if err != nil {
+		log.Fatalf("Error trying to load URL: %q", err)
+	}
+
+	nodes, err := interestNodes(doc, *month, *year)
+	if err != nil {
+		log.Fatalf("Error trying to find link nodes of interest: %q", err)
+	}
+
+	for typ, url := range mapLinks(nodes, *month, *year) {
+		if err = save(typ, url); err != nil {
+			log.Fatalf("Error trying to save file: %q", err)
+		}
+	}
+}
+
+//Load HTML document from specified URL.
```

Comment:
// loadURL loads the HTML document from the specified URL, parses it and returns the root node of the document.

Documentação em Go segue um formato. A documentação é mais usada para membros exportados, porém, fazendo direito estamos tornando a exportação mais tranquila.